configs/raspberrypi_config

```diff
@@ -78,25 +83,40 @@ function platform_modify() {
         run_and_log_cmd "sudo cp work/boot/${RASPBERRYPI_KERNEL_IMAGE} work/rootfs/boot/${MENDER_KERNEL_IMAGETYPE}"
     fi
 
+    # Make sure the on-board bootloader boots our bootloader (which we do via the kernel option).
+    sed -i -e '/^kernel=/d' work/boot/config.txt
+    echo "kernel=${RASPBERRYPI_BOOTLOADER_IMAGE}" >> work/boot/config.txt
```

Comment:
```suggestion
    sed -i -e "s|^kernel=.*|kernel=${RASPBERRYPI_BOOTLOADER_IMAGE}|" work/boot/config.txt
```
I am assuming that there is a `kernel=something` only on the line, i.e.: `.*` is enough, and we do not have to do other funny stuff.

Reply:
`kernel=` can occur multiple times in the file, because it can be listed under each `[rpi3]` / `[rpi4]` tag. The idea behind this is to delete all of them, and add exactly one at the bottom, which is under `[all]`. While this is not completely ideal either (the file we point to is actually not cross-rpi-compatible), it is less confusing than having multiple entries where most of them point to the wrong file.